Intermittent test failure in keystore_test

```
--- FAIL: TestWalletNotifications (0.59s)
    keystore_test.go:344: wallet list doesn't match required accounts: have 541, want 540
    keystore_test.go:372: can't find event with Kind=0 for a91e42fdafd05897598dd5a28219a125ffde0218
FAIL
```

--- a/accounts/keystore/keystore_test.go
+++ b/accounts/keystore/keystore_test.go
@@ -355,7 +355,9 @@ func TestWalletNotifications(t *testing.T) {
 
 	// Shut down the event collector and check events.
 	sub.Unsubscribe()
-	<-updates
+	for ev := range updates {
+		events = append(events, walletEvent{ev, ev.Wallet.Accounts()[0]})
+	}
 	checkAccounts(t, live, ks.Wallets())
 	checkEvents(t, wantEvents, events)
 }

